e2e/test/scenarios/embedding/embed-resource-downloads.cy.spec.ts

```diff
@@ -362,16 +368,32 @@ H.describeWithSnowplowEE(
 
           waitLoading();
 
-          H.main().findByText(value).should("exist");
+          const FIRST_ROW = [
+            5,
+            "leffler.dominique@hotmail.com",
+            FILTER_VALUES[0],
+          ];
 
-          cy.findByRole("button", { name: "Download results" }).click();
-
-          H.popover().within(() => {
-            cy.findByText(".csv").click();
-            cy.findByTestId("download-results-button").click();
+          H.assertTableData({
```

Comment:
assert both the first line of the table, and the first line of the CSV data, that it matches. You can verify this by reverting the fixed code (1 LoC) and the test should fail.